tools/hangar/__snapshots__/test_corpus/valid/struct_json_conversion.w_compile_tf-aws.md

```diff
@@ -0,0 +1,794 @@
+# [struct_json_conversion.w](../../../../../examples/tests/valid/struct_json_conversion.w) | compile | tf-aws
+
+## Advisor.Struct.js
+```js
+module.exports = function(validator_req, fromInline) {
+  class Advisor {
+    static _schema = {
+      id: "/Advisor",
+      type: "object",
+      properties: {
+        ...require("./Person.Struct.js")()._schema.properties,
```

Comment:
I'm curious what's the reason we can't inline all of the properties / constraints into a single schema? Wouldn't that generated code be easier to debug (and run faster, since there's no function calls?)

Reply:
Not a super strong reason, I just opted for the function calls to avoid having to do a bunch of env lookups in the jsify methods. Looking up each extends and any fields that were structs felt heavy, and personally I didnt think the requires were too terrible to follow. (but I have been looking at this for a bit so im biased 😅 )